org.eclipse.lsp4e/plugin.properties

```diff
@@ -25,5 +25,7 @@ notification.category.label = LSP
 notification.event.label = LSP Notification
 command.toggle.highlight.label = Toggle Mark Occurrences
 command.toggle.highlight.name = Toggle Highlight
+command.toggle.outline.sort.name = Sort
```

Comment:
change intended?

Reply:
Usually such command's tool tip says just "Sort", if you mean the change from "Sort Outline" to "Sort" label text then yes, it's intended. Externalization - yes, for sure